packages/fxa-auth-server/lib/routes/totp.js

```diff
@@ -140,6 +140,10 @@ module.exports = (log, db, mailer, customs, config) => {
 
         await db.deleteTotpToken(uid);
 
+        // Downgrade the session to email-based verification when TOTP is
+        // removed. #5154
+        await db.verifyTokensWithMethod(sessionToken.id, 'email-2fa');
```

Comment:
Do we need to updated other sessions besides the current one?

Reply:
I think we should downgrade all sessions that have `totp-2fa`, yes. Although I FWIW I wouldn't be too upset to do that i a followup if we need to get this out urgently for AMO.

Reply:
Good catch! I'll do that in a followup and land this tonight